src/PerfView/SupportFiles/UsersGuide.htm

```diff
@@ -4028,6 +4028,11 @@ <h3>Advanced Options</h3>
             [MethodImpl(MethodImplOptions.AggressiveInlining)], etc.). This feature can also be
             activated with the /JITInlining command line option.
         </li>
+        <li>
+            The <strong><a id="CCWRefCountCheckBox">CCW References Count Checkbox</a></strong> - Causes
```

Comment:
Say .NET Native CCW. and explain that it is only for .NET Native applications.

Reply:
done